Hours.java

```diff
@@ -1,13 +1,26 @@
 class Hours extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * Converting hours into days and hours
+  * @author: Alina Meng
   */
   
   public void run() {
     
-    // start coding here
+    // Variables
+    int intHours;
+    int intDays;
+    int intRemainHours;
+
+    // User input
+    intHours = readInt("How many hours? ");
+    
+    // Caluclations
+    intDays = (int)((double)intHours/24);
```

Comment:
double is unnecissary because it rounds down anyway and all values are integers...